app/config/messaging.js

```diff
@@ -0,0 +1,53 @@
+const joi = require('joi')
+const { PRODUCTION } = require('../constants/environments')
+
+const schema = joi.object({
+  messageQueue: {
+    host: joi.string().default('localhost'),
+    useCredentialChain: joi.bool().default(false),
+    type: joi.string(),
+    appInsights: joi.object(),
+    username: joi.string(),
+    password: joi.string()
+  },
+  sendTopic: {
+    address: joi.string()
+  },
+  eventsTopic: {
+    address: joi.string()
+  }
+})
+
+const config = {
+  messageQueue: {
+    host: process.env.MESSAGE_QUEUE_HOST,
+    useCredentialChain: process.env.NODE_ENV === PRODUCTION,
+    type: 'Topic',
```

Comment:
No need to specify topic.  Only subscriptions need a type.

You could leave it in if you think it makes it more clear, but given the variable is called `eventsTopic` it's probably redundant.